keyboards/nopunin10did/railroad/info.json

```diff
@@ -0,0 +1,125 @@
+{
+    "keyboard_name": "The Railroad",
+    "url": "https://github.com/qmk/qmk_firmware/tree/master/keyboards/nopunin10did/railroad",
+    "maintainer": "NoPunIn10Did",
+    "manufacturer": "NoPunIn10Did",
+    "width": 23.5,
+    "height": 5.25,
+    "key_count": 97,
+    
+    "layouts": {
```

Comment:
```suggestion
    "layouts": {
```

Reply:
Why was key_count removed? I thought I saw that in the QMK guide specifically called out as something to include.

Reply:
https://beta.docs.qmk.fm/developing-qmk/qmk-reference/reference_info_json
It specifically states that key_count is required.

Reply:
That doc is outdated; `key_count` can be determined by just parsing the file, so that key does not need to be explicitly set.